MyOwnORM/MyOwnORM/Helpers/CustomDbSetReflectionHelper.cs

```diff
@@ -0,0 +1,327 @@
+                }
+            }
+
+            return strBuilder.ToString();
+        }
+        public string MapEntityPropertyValuesInString(T obj, string[] names)
+        {
+            Type type = obj.GetType();
+
+            PropertyInfo[] properties = type.GetProperties();
+            StringBuilder strBuilder = new StringBuilder();
+
+            for (int i = 0; i < properties.Length; i++)
+            {
+                bool isCollectionOrModel = false;
+                PropertyInfo prop = properties[i];
+
+                foreach (string name in names)
+                {
+                    if (prop.Name == name)
+                    {
+                        isCollectionOrModel = true;
+                        break;
+                    }
+                }
+                if (isCollectionOrModel == true)
+                    continue;
+
+                object value = prop.GetValue(obj);
+
+                if (prop.PropertyType == typeof(string))
+                {
+                    strBuilder.Append($"'{value}'");
+                }
+                else
+                {
+                    strBuilder.Append(value);
+                }
+
+                if (i < properties.Length - 1)
+                {
+                    strBuilder.Append(", ");
+                }
+            }
+
+            if (strBuilder.ToString().EndsWith(' '))
+            {
+                strBuilder.Length = strBuilder.Length - 2;
+                return strBuilder.ToString();
+            }
+
+            return strBuilder.ToString();
+        }
+
+        public string MapEntityPropertyValuesInUpdateString(T obj, string idProperty)
+        {
+            Type type = obj.GetType();
+
+            PropertyInfo[] properties = type.GetProperties();
+            StringBuilder strBuilder = new StringBuilder();
+
+            for (int i = 0; i < properties.Length; i++)
+            {
+                PropertyInfo prop = properties[i];
+                string columnName = prop.Name;
+                object value = prop.GetValue(obj);
+
+                if (columnName == idProperty) continue;
+
+                if (prop.PropertyType.IsGenericType || prop.PropertyType.IsClass && prop.PropertyType != typeof(string)) continue;
+
+                if (prop.PropertyType == typeof(string))
+                {
+                    strBuilder.Append($"{columnName}='{value}'");
+                }
+                else
+                {
+                    strBuilder.Append($"{columnName}={value}");
+                }
+
+                if (i < properties.Length - 1)
+                {
+                    strBuilder.Append(", ");
+                }
+            }
+
+            if (strBuilder.ToString().EndsWith(' '))
+            {
+                strBuilder.Length = strBuilder.Length - 2;
+            }
+
+            return strBuilder.ToString();
+        }
+
+        public string MapEntityPropertyValuesInUpdateString(dynamic obj, string idProperty)
```

Comment:
Now let's change dynamic as we discussed then(try to put generic, if you don't know type in compilation time, set object but not dynamic). Change similar moment in whole application.